### DISCUSSION TITLE
Couple questions after successfully retrieving inbox.

### DISCUSSION THREAD

**Discussion by joerocks97** [O365/python-o365#815] [Q&A]
#!/usr/bin/python

from O365 import Account

credentials = ('id','secret') 

the default protocol will be Microsoft Graph

account = Account(credentials, auth_flow_type='credentials', tenant_id='xx-xx') 
if account.authenticate():
   print('Authenticated!')
mailbox=account.mailbox(resource='utility@domain.com') 

inbox = mailbox.inbox_folder() 

for message in inbox.get_messages(): 
    print(message) 

Using the code above, was able to retrieve all e-mail from any account in the tenant.  

I would like to use POP to download messages and not just the subject.  The above code list all messages in inbox, but no body, and does not move, but just copies the messages.

Also, in Azure, how to control what mailboxes these App has access to view?  It looks like it by default can view any mailbox in the tenant that is after granted permissions using application and not delegated. 

Thanks for the Python module, it looks very promising since MS is phasing out the legacy protocols except for SMTH.

**Comment by alejcas** [comment]
There are a lot of questions here... POP is not allowed here. We are just using MS Graph api.
The body is included in message.body.

It moves messages using message.move.
The mailbox access is defined by the app and the signin process. Check docs.
